packages/block-editor/src/components/block-lock/toolbar.js

```diff
@@ -33,14 +33,15 @@ export default function BlockLockToolbar( { clientId } ) {
 		}
 	}, [ isLocked ] );
 
-	if ( ! canLock || ( ! isLocked && ! hasLockButtonShown.current ) ) {
+	if ( ! isLocked && ! hasLockButtonShown.current ) {
 		return null;
 	}
 
 	return (
 		<>
 			<ToolbarGroup className="block-editor-block-lock-toolbar">
 				<ToolbarButton
+					disabled={ ! canLock }
```

Comment:
@jeryj Is this accessible? Looks like the Button component was changed to make isFocusable possibly true by default?

Reply:
`Button` has not yet been updated. If we want to disable the button while keeping it focusable, we can add the `__experimentalIsFocusable` prop
```suggestion
					disabled={ ! canLock }
					__experimentalIsFocusable
```

Reply:
The main block icon does the same thing - it's disabled and not focussable.

Reply:
I think we should. If we're going to show a button visually, it needs to be accessible. I think it's time to change that prop in the Button component to just default true. Could discuss that in an issue though.

Reply:
I can access it in VoiceOver using the Shift+control+option+arrows, which announces it as "Unlock, dimmed dialog popup collapsed," but it's not within the arrow keypress roving tabindex. Happy to change it to be a part of the roving tabindex though! Sorry if I got this one wrong! Happy to learn and correct it.

I think we can also use the `[accessibleWhenDisabled](https://ariakit.org/reference/toolbar-item#accessiblewhendisabled)` prop to leave it in the roving tabindex.

Which route would you recommend?

Reply:
@jeryj It's fine, it just goes to show you how Voiceover and Windows accessibility differs greatly. Either way sounds like it should work. I'll let you decide.

Reply:
I think adding it to the roving tabindex with `accessibleWhenDisabled` or `__experimentalIsFocusable` would be a good way to go. It's too easy to miss the information otherwise.

Reply:
`__experimentalIsFocusable` comes from the `Button` component from `@wordpress/components`, `accessibleWhenDisabled` comes from `ariakit` — I wouldn't know in advance which approach would work better. Could you try both and see if there are any differences?

Reply:
I tested it out. `accessibleWhenDisabled` works well with the toolbar. Using `__experimentalIsFocusable` doesn't work out of the box. It puts it at the end of the toolbar roving tabindex order or skips it altogether for some reason.